modules/ROOT/pages/about-flows.adoc

```diff
@@ -42,20 +37,27 @@ out to other services, you might factor out that behavior into other flows.
 
 == Using Subflows
 
-You can also group a series of event processors into subflows. Like a flow, a
-subflow is a scope that groups together a sequence of event processors, but it
-does not have an event source, and it does not have its own error handling.
-Subflows are always called from another flow or subflow through a Flow Ref
-component.
+A subflow is a scope that enables you to group event processors, similar to a flow, but with certain differences and limitations:
+
+* Subflows do not have event sources.
+* Subflows do not have an error handling scope.
+* During design, subflows work as a macro that replaces the Flow Reference components that call the subflow. When you build the application, all Flow Reference components that call a subflow are replaced by the contents of the referenced subflow.
+* Referencing a subflow has higher performance than referencing a flow.
+
+=== Considerations
```

Comment:
I've been taught that "Considerations" is an unnecessarily vague heading that can typically replaced by what we REALLY mean by that broad term: Requirements? Restrictions? Limitations? 
In this particular case, you could probably just add this text after the list in the previous section.